vowpalwabbit/slim/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+project(vwslim CXX)
+set(CMAKE_CXX_STANDARD 11)
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/include ${CMAKE_CURRENT_SOURCE_DIR}/../ ${CMAKE_CURRENT_SOURCE_DIR}/../../explore)
```

Comment:
Please use new style CMake, these commands usually take the form of target_... so this should be `target_include_directories`

Reply:
Fixed

---
In reply to: [267927043](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1800#discussion_r267927043) [](ancestors = 267927043)